lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -219,6 +205,15 @@
         }
       } // EditorHas.functionBar
 
+      if (ActiveOptions['section'] && $.markupSupportsEditableSections ( ActiveOptions.MarkupType )) {
+        var sectionLine = $.findSection(ActiveOptions['section'], LanguageDefinition.getDefinitionFor('gollum-helpers'));
+        if (sectionLine) {
+          window.ace_editor.gotoLine(sectionLine+1, offset=0, slap=false); //gotoLine starts counting at 1 instead of 0, so +1
+          window.ace_editor.scrollToLine(sectionLine, false, false);
```

Comment:
```javascript
window.ace_editor.scrollToLine(sectionLine, center=false, animate=false);
```
Why scrollToLine here instead of goToLine?

Reply:
we need both:
* goto moves the cursor
* scroll to changes the editor focus